hw2/SUNGOD3/q2/pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = --ignore=extern/pybind11/tests
```

Comment:
Could you please remove this file? I guess that you refer the pr #76 to finish your homework, but this file is meaningless when you don't set any `.gitmodules` in your homework.